proposals/wac-ucr/index.bs

```diff
@@ -608,6 +827,15 @@ be immediately visible. Other posts should only be visible and editable to
 those who wrote them. They can then be viewable to the world when they get
 reviewed.
 
+<div class="assertion">
+
+* [[#req-inheritance-default-permissions]]
+* [[#req-agent-group]]
+* [[#req-conditional-relationship]]
+
```

Comment:
But in this particular use case we're not saying that we are presenting a credential. If we wanted to add that requirement, the use case should also be updated.

Reply:
I think of WebIDs as credentials to. Think of them as self signed if you want. The Credential document is your WebID Profile. You can think of it as being signed by your `https` enabled web server, serving the definition of the WebID, which can specify your key. 
 
See the [Section on Linking your WebID to a key](https://github.com/bblfish/authentication-panel/blob/HttpSig/HttpSignature.md#linking-a-webkey-to-a-webid) in my extension of HTTP Signatures authentication.

Reply:
I don't disagree with what you're saying. My point is that the proposal was to add the credential determination requirement to this use case, and in the case of a webid you don't really need that requirement.

Reply:
The use case is"[Alice now] would like to try a compromise: allow the posts from her extended social network (friend of her friends, colleagues and family) to be immediately visible."

The requirement is:
"The system shall ensure that there are practical and efficient mechanism available for the client to determine an appropriate credential to present for access to a given resource."

Without that the friends of Alice would have no way of knowing how to authenticate. Should they use their Job IDs? Their University IDs, Age Credentials? They can't know. Perhaps they have a credential signed by a friend! Could work too.

It can work using OIDC or [using HTTP Signature](https://github.com/bblfish/authentication-panel/blob/HttpSig/HttpSignature.md). But a proof has to be given of that relation, which is believable - credible - ie a Credential. 


client has to present a WebID and a key to auth

Reply:
Added vc reqs to the use case in https://github.com/solid/authorization-panel/pull/152/commits/0185b4da4a38a15daeab7ee6dde730f6b409e530